src/python_tools/BUILD

```diff
@@ -79,3 +79,23 @@ py_binary(
     srcs = ["robot_broadcast_sender.py"],
     deps = ["//proto:announcement_py_proto"],
 )
+
+py_binary(
+    name = "rotary_encoder",
+    srcs = ["../src/software/jetson_nano/rotary_encoder/rotary_encoder.py"],
+    deps = [requirement("Jetson.GPIO")],
+)
+
+py_binary(
+    name = "lcd_user_interface",
+    srcs = ["../src/software/jetson_nano/lcd_user_inteface/lcd_user_inteface.py"],
+    deps = [
+        requirement("Adafruit-Blinka"),
+        requirement("adafruit-circuitpython-busdevice"),
+        requirement("adafruit-circuitpython-rgb-display"),
+        requirement("Adafruit-PlatformDetect"),
+        requirement("Adafruit-PureIO"),
+        requirement("Jetson.GPIO"),
+        requirement("Pillow"),
+    ],
+)
```

Comment:
you should move this to `software/jetson_nano/display`

Reply:
I created a new `BUILD` file in the `software/jetson_nano/display/` folder. CI is failing because with the following error message: 
(22:13:18) ERROR: /home/runner/work/Software/Software/src/software/jetson_nano/display/BUILD:5:10: no such package '@python_tools_deps//pypi__jetson_gpio': BUILD file not found in directory 'pypi__jetson_gpio' of external repository @python_tools_deps. Add a BUILD file to a directory to mark it as a package. and referenced by '//software/jetson_nano/display:rotary_encoder'

Reply:
Ah i forgot to mention, you also need to add this to the WORKSPACE file

~~~
 pip_install(                                                                                                                                                 
     name = "display_deps",                                                                                                                                   
     python_interpreter_target = "@python_interpreter//:python_bin",                                                                                          
     requirements = "//software/jetson_nano/display:requirements.txt"                                                                                         
 )                                        
~~~

and then update the top of the BUILD file to point to @display_deps and not @python_tools

Reply:
Thanks, that worked. Fixed now.